docs/deployment/pixi_pack.md

```diff
@@ -56,9 +56,42 @@ Since `pixi-pack` just downloads the `.conda` and `.tar.bz2` files from the cond
 pixi-pack pack --platform win-64
 ```
 
-!!!note ""
+!!! note
     You can only unpack a pack on a system that has the same platform as the pack was created for.
 
+### Self-Extracting Binaries
+
+You can create a self-extracting binary that contains the packed environment and a script that unpacks the environment.
+This can be useful if you want to distribute the environment to users that don't have `pixi-pack` installed.
+
+```bash
+# unix
+$ pixi-pack pack --create-executable
+$ ls
+environment.sh
+$ ./environment.sh
+$ ls
+env/
+activate.sh
+environment.sh
+```
+
+```powershell
+# windows
+PS > pixi-pack pack --create-executable
+PS > ls
+environment.ps1
+PS > .\environment.ps1
+PS > ls
+env/
+activate.sh
+environment.ps1
+```
```

Comment:
We can make this toggleable using `===` (see other examples in the docs)

Reply:
Good idea, done